rs/execution_environment/src/execution_environment/tests.rs

```diff
@@ -718,7 +718,7 @@ fn get_canister_status_from_another_canister_when_memory_low() {
     let controller = test.universal_canister().unwrap();
     let binary = wat::parse_str("(module)").unwrap();
     let canister = test.create_canister(Cycles::new(1_000_000_000_000));
-    let memory_allocation = NumBytes::from(300);
+    let memory_allocation = NumBytes::from(450);
```

Comment:
This is changed since the test was failing on `debug_assert!` with message 
`Used non-Wasm memory: 412 is larger than memory allocation: 300.`